en/SUMMARY.md

```diff
@@ -202,10 +202,11 @@
       * [Holybro pix32 (FMUv2)](flight_controller/holybro_pix32.md)
       * [mRobotics-X2.1 (FMUv2)](flight_controller/mro_x2.1.md)
       * [mRo Control Zero F7)](flight_controller/mro_control_zero_f7.md)
-      * [NXP RDDRONE-FMUK66 FMU](flight_controller/nxp_rddrone_fmuk66.md)
       * [ModalAI Flight Core v1](flight_controller/modalai_fc_v1.md)
       * [ModalAI VOXL Flight](flight_controller/modalai_voxl_flight.md)
       * [ModalAI VOXL 2](flight_controller/modalai_voxl_2.md)
+      * [NXP RDDRONE-FMUK66 FMU](flight_controller/nxp_rddrone_fmuk66.md)
+      * [Sky-Drones AIRLink](flight_controller/airlink.md)
```

Comment:
FYI, all names are Manufacturer-FC name, alphabetical order

Reply:
Apologies and thanks for fixing!